### ISSUE TITLE
Support Fluentd in_tail plugin 

### ISSUE THREAD

**Issue by karan56625** [fluent/fluent-operator#829]
### Is your feature request related to a problem? Please describe.

In order to meet our organization's requirements, we need to utilize Fluentd only mode in Fluent-operator, similar to our current usage of Fluentbit only mode. However, there are certain essential Fluentd inputs that are currently unavailable to accomplish this transition. One such crucial input that is missing is the systemd input

### Describe the solution you'd like

We are requesting the inclusion of Fluentd Systemd Input support within the fluent-operator. Since we have already introduced support for the Fluentd tail plugin in the fluent-operator, we would like to extend similar support for Systemd as well.

**Comment by karan56625** [comment]
> Now the fluentd controller will only create a fluentd statefulset for now, it's only applicable to receive logs from network.
> For systemd input, maybe you'll need a daemonset

yes, I observed we have already enhancement task for that https://github.com/fluent/fluent-operator/issues/755 . I will be sharing PR for this soon.

**Comment by joshuabaird** [comment]
This issue is being closed due to inactivity. Please re-open if it is still relevant.
